dashboard/dashboard.html

```diff
@@ -980,6 +980,7 @@
 	logSegment.appendChild(h("div", attrs, [
 		h('span', null, " IGNOR "),
 		h('a', {"href": data["url"], "className": "ignore"}, data["url"]),
+		data["source"],
```

Comment:
I think I prefer this modification instead:

```diff
        var attrs = Reusable.obj_className_line_ignore;
        logSegment.appendChild(h("div", attrs, [
                h('span', null, " IGNOR "),
+               "(" + data["source"] + "): ",
                h('a', {"href": data["url"], "className": "ignore"}, data["url"]),
-               data["source"],
                h('span', Reusable.obj_className_bold, " by "),
                data["pattern"]
        ]));
```

This produces:

![tumblrignore1](https://cloud.githubusercontent.com/assets/3859/12371283/4d8c405e-bbf3-11e5-850e-9011da610c43.png)

The diff in this PR requires you to horizontally scroll and has no whitespace between the URL and the source:

![tumblrignore2](https://cloud.githubusercontent.com/assets/3859/12371285/5eff4c3c-bbf3-11e5-9d51-5f12ade290a7.png)